apps/webapp/app/routes/_app.orgs.new/route.tsx

```diff
@@ -55,6 +59,67 @@ export const action: ActionFunction = async ({ request }) => {
       companySize: submission.value.companySize ?? null,
     });
 
+    if (env.PLAIN_API_KEY) {
+      const client = new PlainClient({
```

Comment:
Create a new `plain.server.ts` file, maybe in the webapp utilities folder.

It would be:

```ts
export async function sendToPlain({ userId, email, orgName, title, components}: Input) {
  const client = new ......
}
```

Then in this particular case you would wrap this in `try/catch` so it doesn't stop people creating orgs if Plain is down.